common/utilities/mdc.py

```diff
@@ -0,0 +1,21 @@
+import contextvars
+
+from comms.http_headers import HttpHeaders
+
+message_id: contextvars.ContextVar[str] = contextvars.ContextVar('message_id', default='')
+correlation_id: contextvars.ContextVar[str] = contextvars.ContextVar('correlation_id', default='')
+inbound_message_id: contextvars.ContextVar[str] = contextvars.ContextVar('inbound_message_id', default='')
+interaction_id: contextvars.ContextVar[str] = contextvars.ContextVar('interaction_id', default='')
+
+
+def build_tracking_headers():
+    headers = {}
+    if correlation_id.get():
+        headers[HttpHeaders.CORRELATION_ID] = correlation_id.get()
+    if message_id.get():
+        headers[HttpHeaders.MESSAGE_ID] = message_id.get()
+    if interaction_id.get():
+        headers[HttpHeaders.INTERACTION_ID] = interaction_id.get()
+    if inbound_message_id.get():
+        headers[HttpHeaders.INBOUND_MESSAGE_ID] = inbound_message_id.get()
```

Comment:
If you don't set keys in header, trying to access `headers[HttpHeaders.CORRELATION_ID]` will rise `KeyError`. Maybe you should put all keys inside `header` but make their default value as `None` or empty string. That way calling `headers[HttpHeaders.CORRELATION_ID].get()` will equal to `None` or empty string

Reply:
If I leave key with empty value, I assume it's exactly what will be added to HTTP headers - empty header. I think it's not something we want.
The headers builder is used only for HTTP requests.

Reply:
ok